tests/e2e/lib/backup.go

```diff
@@ -25,9 +24,6 @@ func CreateBackupForNamespaces(ocClient client.Client, veleroNamespace, backupNa
 			SnapshotMoveData:         &snapshotMoveData,
 		},
 	}
-	if snapshotVolumes {
```

Comment:
@mrnold I reverted some changes from your PR that I think were not necessary (this option for example, is enabled by default)

Checking if the kubevirt test pass is enough to assure I did not break anything there?

Reply:
Yes, that's fine. I wasn't 100% sure whether or not this was needed.